packages/framework/src/Model/Customer/UserFactory.php

```diff
@@ -11,27 +11,48 @@ class UserFactory implements UserFactoryInterface
      */
     protected $entityNameResolver;
 
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Customer\CustomerPasswordService
+     */
+    private $customerPasswordService;
+
     /**
      * @param \Shopsys\FrameworkBundle\Component\EntityExtension\EntityNameResolver $entityNameResolver
+     * @param \Shopsys\FrameworkBundle\Model\Customer\CustomerPasswordService $customerPasswordService
      */
-    public function __construct(EntityNameResolver $entityNameResolver)
+    public function __construct(EntityNameResolver $entityNameResolver, CustomerPasswordService $customerPasswordService)
```

Comment:
Again, I did not notice that in the previous commits, however, I believe the change of constructor interface should be mentioned in `UPGRADE.md`. Like I did when we were refactoring pricing...https://github.com/shopsys/shopsys/blame/master/UPGRADE.md#L95

Reply:
Maybe it could be a better idea to write upgrading instruction after the whole refactoring is finished :thinking: